### PR TITLE
Make addon Ember 2 compatible

### PR DESCRIPTION
**Pull request by dgavey**
This just updates the container injection to be 1 or 2 compliant and forces the use of https on the calls.

### CODE REVIEW

File: app/kmqs/main.js [hunk 1]
```diff
@@ -16,8 +16,8 @@ function load(u) {
 }
 
 if (config.kissmetricsKey) {
-  load('//i.kissmetrics.com/i.js');
-  load('//doug1izaerwt3.cloudfront.net/' + config.kissmetricsKey + '.1.js')
+  load('https://i.kissmetrics.com/i.js');
```
Comment (stefanpenner) [inline app/kmqs/main.js]: do they no longer offer http? or..

This seems unrelated to ember 2?

Comment (dgavey) [inline app/kmqs/main.js]: @stefanpenner it is unrelated, sorry I should have done two PR's.  We had to force it to use https:// because of a limitation in Cordova.  If you'd prefer I can put it behind a flag of some sort to force SSL on.

Comment (stefanpenner) [inline app/kmqs/main.js]: well, i suspect it could detect if it is `file://` like cordova and then force https automatically.

Comment (dgavey) [inline app/kmqs/main.js]: Unfortunately it doesn't, it tries to load the files from the local system. (edit, I read you're comment wrong, yeah I can try and do an autodetect, that sounds like a plan)

**Comment by dgavey** [comment]
Couldn't figure out an easy way to detect of the page was being loaded from file:// so I opted for a config var that forces SSL loading.  I could have detected Cordova but I feel this method is probably more adaptable for other potential cases where someone might want to force SSL.

File: app/initializers/ember-kiss-metrics.js [hunk 2]
```diff
@@ -1,5 +1,10 @@
-export function initialize(container, application) {
-  container.options('kmq:main');
+export function initialize() {
+  let application = arguments[1] || arguments[0];
```
Comment (stefanpenner) [inline app/initializers/ember-kiss-metrics.js]: sorry to be pedantic, but I believe a more accurate check is arity? `arguments.length === 1` vs `arguments.length === 2`

File: app/kmqs/main.js [hunk 3]
```diff
@@ -16,8 +16,13 @@ function load(u) {
 }
 
 if (config.kissmetricsKey) {
-  load('//i.kissmetrics.com/i.js');
-  load('//doug1izaerwt3.cloudfront.net/' + config.kissmetricsKey + '.1.js')
+  let pathPrefix = '//';
+  if (config.kissmetricsForceSSL) {
```
Comment (stefanpenner) [inline app/kmqs/main.js:20]: looks good

**Comment by stefanpenner** [comment]
1 nitpick, then we can :shipit:

**Comment by dgavey** [comment]
Ok it's updated.

**Comment by stefanpenner** [comment]
awesome thanks + released :)

```
➜  ember-kiss-metrics git:(master) git push origin v1.0.0
Total 0 (delta 0), reused 0 (delta 0)
To git@github.com:stefanpenner/ember-kiss-metrics.git
 * [new tag]         v1.0.0 -> v1.0.0
➜  ember-kiss-metrics git:(master) npm publish
+ ember-kiss-metrics@1.0.0
```

**Comment by dgavey** [comment]
Great, thanks a bunch!
